cirq-core/cirq/experiments/t1_decay_experiment_test.py

```diff
@@ -228,6 +202,39 @@ def test_curve_fit_plot_warning():
         bad_fit.plot(include_fit=True)
 
 
+@pytest.mark.parametrize('t1', [200, 500, 700])
+def test_noise_model_constant(t1):
+    class GradualDecay(cirq.NoiseModel):
```

Comment:
I think using `cirq.NoiseModel.from_noise_model_like(cirq.amplitude_damp(...))` may provide a stronger demonstration here, since the new simulator-based experiment also works with duration-agnostic noise models.